src/xclim/sdba/_adjustment.py

```diff
@@ -587,6 +587,13 @@ def qdm_adjust(ds: xr.Dataset, *, group, interp, extrapolation, kind) -> xr.Data
             sim : Data to adjust.
     """
     sim_q = group.apply(u.rank, ds.sim, main_only=True, pct=True)
+    if group.prop and interp != "nearest":
+        import logging
+
+        logging.getLogger("xclim").warning(
+            f"Using a {interp} interrpolation with QuantileDeltaMapping might create sudden jumps between different"
+            " groups. See discussion https://github.com/Ouranosinc/xclim/discussions/2048 for more information "
+        )
```

Comment:
Looks good!

I think the `warnings` library is a better fit here: 
```suggestion
        msg = (
            f"Using a {interp} interrpolation with QuantileDeltaMapping might create sudden jumps between different"
            " groups. See discussion https://github.com/Ouranosinc/xclim/discussions/2048 for more information."
        )
        warnings.warn(msg)
```